_includes/markdown/Markup.md

```diff
@@ -257,6 +257,21 @@ ARIA attributes can be added with JavaScript to help dynamically add context to
 
 You can see how effortless it is to make our tabbed interface accessible to screen readers. All we need to do is add context.
 
+#### Skip to Content Links
+
+Skip to content links, also known as "skip links", allow screen readers and assistive technologies to skip directly to content. Skip links are a WordPress Core standard that 10up aims to implement across all projects.
```

Comment:
I've edited it where there's an additional link. I wanted to keep the one at the end as it links straight to the theme handbook and shows the CSS for it wheres the one I added is more of a general definition for those who aren't familiar with them.